tests/test_timestream.py

```diff
@@ -19,16 +20,32 @@ def test_write_simple():
     )
     rejected_records = wr.timestream.write(
         df=df,
-        database="sampleDB",
-        table="sampleTable",
+        database=name,
+        table=name,
         time_col="time",
         measure_col="measure",
         dimensions_cols=["dim0", "dim1"],
     )
     assert len(rejected_records) == 0
+    df = wr.timestream.query(f"""
+        SELECT
+            1 as col_int,
+            try_cast(now() as time) as col_time,
+            TRUE as col_bool,
+            current_date as col_date,
+            'foo' as col_str,
+            measure_value::double,
+            measure_name,
+            time
+        FROM "{name}"."{name}"
+        ORDER BY time
+        DESC LIMIT 10
+    """)
+    assert df.shape == (3, 8)
 
 
-def test_write_real():
+def test_advanced(timestream_database_and_table):
```

Comment:
nit: you could rename the method name to a better descriptive name

Reply:
Done! Also added a count check for the `test_real_csv_load_scenario` using the `wr.timestream.query()`.